ee/clickhouse/migrations/0027_persons_and_groups_on_events.py

```diff
@@ -0,0 +1,35 @@
+from infi.clickhouse_orm import migrations
```

Comment:
@fuziontech I only need your 👀 on this file. Notice the `ON CLUSTER` when updating `events`, `writable_events`, and `sharded_events`